fiftyone/core/annotation/generate_label_schema.py

```
@@ -0,0 +1,436 @@
+    fields = _flatten_fields(sample_collection, fields)
+
+    schema = {}
+    for field_name in fields:
+        schema[field_name] = _generate_field_label_schema(
+            sample_collection, field_name, scan_samples
+        )
+
+    return schema
+
+
+def _flatten_fields(collection, fields):
```

Ruff (0.14.5)

Missing return type annotation for private function `_flatten_fields`

(ANN202)